backend-acceptance-testing/requirements-py3.txt

```diff
@@ -1,6 +1,7 @@
 bravado==9.2.2
 crypto
 cryptography==36.0.1
+docker>=5.0
```

Comment:
This syntax does not play well with `dependabot`, we should use `==` to today's latest.